justfile

```diff
@@ -314,4 +347,11 @@ wipe-prometheus:
     cd services/prometheus
     rm -rf data
 
+
+# end-to-end tests
+e2e: services
+    #!/usr/bin/env bash
+    set -euxo pipefail
+    cargo test -- --test-threads=1
```

Comment:
🔧 I think you want to specify the package here.

Reply:
it got fixed in the subsequent commit where I introduced the package.